app/models/manageiq/providers/hawkular/middleware_manager/refresh_parser.rb

```diff
@@ -94,6 +95,18 @@ def fetch_server_entities
         end
       end
 
+      def fetch_availability
+        metrics_ids = {}
```

Comment:
I'm ok with all the names, was just wondering about the plural part repeated in both "metrics" and "ids". it's really a nit.